platform/ios/src/MGLMapView.mm

```diff
@@ -1676,9 +1718,13 @@ - (void)handleTwoFingerTapGesture:(UITapGestureRecognizer *)twoFingerTap
 
     _mbglMap->cancelTransitions();
 
+    self.cameraChangeReason = MGLCameraChangeReasonGestureTwoFingerSingleTap;
+
     if (twoFingerTap.state == UIGestureRecognizerStateBegan)
     {
         [self trackGestureEvent:MGLEventGestureTwoFingerSingleTap forRecognizer:twoFingerTap];
+
+        [self notifyGestureDidBegin];
```

Comment:
Will need to double check, but from memory (testing on simulator), the Began state branch was never entered anyway - and so may still not be calling the willChange methods 😢.

Thanks for all the great feedback @friedbunny!

Reply:
Yep - the ...Began state is not called - I'm assuming because of the requirements on the pan and friends gestures.

However the later setZoom call will trigger the willChange delegate method, so this is more of an issue for the `[self trackGestureEvent:MGLEventGestureTwoFingerSingleTap forRecognizer:twoFingerTap]` call. I'll add a ticket.